[DOCS-5940] Add Container Images configuration info (#20865)
DataDog/documentation: content/en/infrastructure/containers/configuration.md

However, the scrubber does not scrub paths that contain sensitive words. For example, it does not overwrite `/etc/vaultd/secret/haproxy-crt.pem` with `/etc/vaultd/******/haproxy-crt.pem` even though `secret` is a sensitive word.

> ## Configure Container Images View

Reviewer: ```suggestion
## Configure container images view
```

> ## Configure container images view

Reviewer 2: ```suggestion
## Configure Container Images view
```

> ### Image Collection

Reviewer: ```suggestion
### Image collection
```

> Datadog collects container image metadata to provide enhanced debugging context. 

Reviewer 2: Maybe expand on this a bit?

```suggestion
Datadog collects container image metadata to provide enhanced debugging context for related containers and CSM vulnerabilities.
```

#### Agent

{{< tabs >}}
{{% tab "Kubernetes (Helm)" %}}

> If you are using helm version `>= 3.46.0`, image collection is [enabled by default][3].</br>

Reviewer: ```suggestion
If you are using Helm version `>= 3.46.0`, image collection is [enabled by default][3].</br>
```

Reviewer: The link in this line is broken. Links inside tabs require you to put the link target inside the tab (before the closing tab shortcode).

Reviewer: Just noticed that you did put the link inside the tab. Is it the same one? If so, update the link target to [1] instead of [3].

> If you are using helm version `>= 3.46.0`, image collection is [enabled by default][3].</br>

Reviewer: ```suggestion
If you are using helm version `>= 3.46.0`, image collection is [enabled by default][1].</br>
```

> Or, add the following to your `values.yaml` helm configuration file:

Reviewer: ```suggestion
Or, add the following to your `values.yaml` Helm configuration file:
```

```yaml
datadog:
  containerImageCollection:
    enabled: true
  sbom:
    containerImage:
      enabled: true
```
[1]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L651
{{% /tab %}}

{{% tab "Kubernetes (Operator)" %}}

Add the following to the spec section of your `values.yaml` file:

```yaml
apiVersion: datadoghq.com/v2alpha1
kind: DatadogAgent
metadata:
  name: datadog
spec:
  features:
    # ...
    sbom:
      enabled: true
      containerImage:
        enabled: true
```

{{% /tab %}}

{{% tab "ECS EC2" %}}

> To enable container image vulnerability scanning on your [ECS EC2 instances][4], add the following environment variables to your `datadog-agent` container definition:

Reviewer: Link is broken

Reviewer: This link is still broken. Please fix it by adding the link target **inside** the tab.

```yaml
{
    "containerDefinitions": [
        {
            "name": "datadog-agent",
             ...
            "environment": [
              ...
              {
                "name": "DD_CONTAINER_IMAGE_ENABLED",
                "value": "true"
              },
              {
                "name": "DD_SBOM_ENABLED",
                "value": "true"
              },
              {
                "name": "DD_SBOM_CONTAINER_IMAGE_ENABLED",
                "value": "true"
              }
            ]
        }
    ]
  ...
}
```

If the Agent fails to extract the SBOM from the container image, increase the Agent memory in the container definition:

```yaml
{
    "containerDefinitions": [
        {
            "name": "datadog-agent",
            "memory": 256,
            ...
        }
     ]
    ...
}
```

{{% /tab %}}

{{% tab "Hosts" %}}

Add the following to your `datadog.yaml` configuration file:

```yaml
sbom:
  enabled: true
  container_image:
    enabled: true
container_image:
  enabled: true
```

{{% /tab %}}
{{< /tabs >}}

> #### Container Registries

Reviewer: ```suggestion
#### Container registries
```

> ##### AWS Elastic Container Registry

Reviewer: ```suggestion
##### Amazon Elastic Container Registry (Amazon ECR)
```

> Set up the [AWS integration][5] to begin crawling Container Image metadata from AWS Elastic Container Registry

Reviewer: ```suggestion
Set up the [AWS integration][5] to begin crawling container image metadata from Amazon ECR.
```

[1]: https://app.datadoghq.com/containers
[2]: /infrastructure/containers

> [3]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L651 
> [4]: /containers/amazon_ecs/?tab=awscli#setup

Reviewer: ```suggestion
```
I think these are the links that you'll have to move inside tabs, so you should be able to remove them from the bottom of the file.
